README.md

```diff
@@ -0,0 +1,51 @@
+# Project Name
+
+> This project is a portfolio website which is created using pure Html5 and Css3.
+
+## Built With
+
+- Html
+- Css
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
```

Comment:
**OPTIONAL** - Please include steps on how to clone, navigate and open your project locally, this would help make your repo even more professional. Please fix this 👍 👍